Makefile

```diff
@@ -173,6 +173,20 @@ eunit: couch
 	  $(REBAR) -r eunit $(EUNIT_OPTS) apps=$$dir || exit 1; \
 	done
 
+.PHONY: exunit
```

Comment:
Don't forget Makefile.win, please.

Reply:
Makefile.win is updated (unfortunately I don't have access to windows machine to test it)